testnet/register-node.js

```diff
@@ -226,14 +226,6 @@ function main() {
 
     web3.setProvider(new Web3.providers.HttpProvider(externalConfig.blockchain.rpc_server_url));
 
-    if (!externalConfig.node_wallet ||
-        !externalConfig.node_private_key ||
-        !web3.utils.isAddress(externalConfig.node_wallet)) {
-        logger.error('Wallet not provided! Please provide valid wallet.');
-        process.exit(1);
-        return;
-    }
-
```

Comment:
If we're removing this wallet check we need to somewhere throw an error if a user does not have a wallet for an implementation

Reply:
It is implementation specific now and we should moved wallet check in corresponding blockchain classes